lang/cs/Settings.StyleCop

```diff
@@ -682,146 +682,146 @@ under the License.
       <Rules>
         <Rule Name="KeywordsMustBeSpacedCorrectly">
           <RuleSettings>
-            <BooleanProperty Name="Enabled">False</BooleanProperty>
+            <BooleanProperty Name="Enabled">True</BooleanProperty>
```

Comment:
Is there any default value for Enabled property? It would be nice to be able to just remove RuleSettings section for rules which are enabled, like we do with Checkstyle, if it defaults to True.